projects/flow.groups-2D-random-walk/src/project.py

```diff
@@ -0,0 +1,152 @@
+import flow
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# Helper functions
+def get_first_replica(jobs, first_index=1):
+    try:
+        job = next(filter(lambda job: job.sp.replica == first_index, jobs))
+    except StopIteration:
+        raise RuntimeError("First replica not found.")
+    return job
+
+
+def positions_generator(jobs):
+    for job in jobs:
+        with job.data:
+            yield job.data.positions
+
+
+def aggregate_cond_true(doc_attr):
+    """Create a condition which is true if all jobs have doc_attr as truthy."""
+
+    def true_for_all_docs(*jobs):
+        return all(job.doc.get(doc_attr, False) for job in jobs)
+
+    return true_for_all_docs
+
+
+class Project(flow.FlowProject):
+    """Create a workflow for simulating 2D Gaussian random walks."""
+
+    pass
+
+
+@Project.label
+def simulated(job):
+    return "positions" in job.data
+
+
+def all_simulated(*jobs):
+    return all(simulated(job) for job in jobs)
+
+
+# Create aggregator that combines all replicas with a single standard deviation
+std_aggregator = flow.aggregator.groupby("std")
```

Comment:
Let's rename this variable/state point parameter to `sigma` instead of `std`.

Reply:
Personally I like `std` better because when I see it I know we are talking about standard deviation, `sigma` is less descriptive to me... I guess it really depends on which name people are more used to.